tests/components/samsungtv/test_media_player.py

```diff
@@ -150,6 +150,28 @@ def mock_now():
         assert state.state == STATE_OFF
 
 
+async def test_update_access_denied(hass, remote, mock_now):
+    """Testing update tv unhandled response exception."""
+    with patch(
+        "homeassistant.components.samsungtv.media_player.SamsungRemote",
```

Comment:
Maybe we should move the patch of the media_player Samsung remote down before the update is scheduled to make it clear where we expect the exception to happen?